src/redisai.c

```diff
@@ -456,7 +456,12 @@ int RedisAI_ModelGet_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
 
   if (meta && blob) {
     RedisModule_ReplyWithCString(ctx, "blob");
-    RedisModule_ReplyWithStringBuffer(ctx, buffer, len);
+    const size_t n_chunks = len / RAI_CHUNK_LEN + 1;
+    RedisModule_ReplyWithArray(ctx, (long)n_chunks);
```

Comment:
@lantiga this will break the clients correct? IMO we also need the change documented as soon as we merge this.

Reply:
Yes it will. Part of this review was also to coordinate on this aspect.

Reply:
Done